connectors/http/http-base/src/main/java/io/camunda/connector/http/base/model/HttpCommonRequest.java

```diff
@@ -120,6 +120,15 @@ public class HttpCommonRequest {
       optional = true)
   private String skipEncoding;
 
+  @TemplateProperty(
+      label = "isHeaderGroupingEnabled",
+      description = "Group incoming headers with same name into a List",
```

Comment:
```suggestion
      description = "Group incoming headers with same name into a List to support <a href="https://datatracker.ietf.org/doc/html/rfc6265">multiple Set-Cookie headers.</a>",
```

Reply:
I added this. Where can I see, if this is displayed correctly? Where is this shown for hidden properties?

Reply:
Saw its a hidden property after looking at this. They are not shown (because they are hidden 😄) and will therefore not be shown.